towhee/types/audio_frame.py

```diff
@@ -24,12 +24,13 @@ class AudioFrame(np.ndarray):
             The audio sample rate in Hz.
     """
```

Comment:
Can we add some test cases in docstring so that the code blocks will appear in the API docs?

Reply:
The AudiaoFrames are created by audio_decoder.ffmpeg, we will add an example in decoder op's README.